includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
So that implies that you could create a *Wikia Home (//Corp) Page* (or at least a wiki that looks like one) and it doesn't need to be added to the *Corporate Footer* dropdown. Am I right? And this idea actually appeals to me. :+1: